### DISCUSSION TITLE
WiFi password change - How to update WiFi settings without hard reset ?

### DISCUSSION THREAD

**Discussion by Techbrunch** [Hypfer/Valetudo#2142] [Q&A / Support] [answered]
### Are you already using Valetudo?

Yes, for a long time

### Your questions

Hello,

I just changed my WiFi password and I can't find the process to go back to the setup mode of Valetudo so that I can update the wifi password and SSID.

I'm using a Roborock S50.

**Comment by vivia** [comment, marked as answer]
Long-press the two outer buttons, so that the robot spawns its own access point. Then you can connect to it and reconfigure the WiFi.

**Reply by Techbrunch** [reply]
Thanks for the quick answer !
